test/module/irohad/pending_txs_storage/pending_txs_storage_test.cpp

```diff
@@ -130,6 +129,94 @@ TEST_F(PendingTxsStorageFixture, InsertionTest) {
   }
 }
 
+/**
+ * PaginationMeta works in PendingTxsStorage
+ * @given Batch of two transactions and storage
+ * @when storage receives updated mst state with the batch
+ * @then list of pending transactions beetwen two timestamps can be obtained
+ */
+TEST_F(PendingTxsStorageFixture, TxPaginationTestFirsTimeLastTimeSpecified) {
+  auto state = emptyState();
+  auto first_time = getUniqueTime();
+  auto transactions = twoTransactionsBatch();
+  auto last_time = getUniqueTime();
+  *state += transactions;
+  auto transactions1 = twoTransactionsBatch();
+  *state += transactions1;
+  const auto kPageSize = 100u;
+  Response expected;
+  expected.transactions.insert(expected.transactions.end(),
+                               transactions->transactions().begin(),
+                               transactions->transactions().end());
+  expected.all_transactions_size = transactions->transactions().size();
+  storage_->updatedBatchesHandler(state);
+  const auto &creator = "alice@iroha";
+  auto pending = storage_->getPendingTransactions(creator, kPageSize, std::nullopt, first_time, last_time);
+  IROHA_ASSERT_RESULT_VALUE(pending);
+  ASSERT_EQ(expected.all_transactions_size, pending.assumeValue().transactions.size() );
+}
```

Comment:
I suggest white lines to separate lines of code not connected that much. E.g. between checks.
I suggest to look into rest of tests - there are white lines to group more connected lines of code